### PR TITLE
Feature sum

### PR DESCRIPTION
**Pull request by michaeljclausen**
Here it is!

### CODE REVIEW

File: client/js/table-model.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 class TableModel {
-  constructor(numCols=10, numRows=20) {
+  constructor(numCols=4, numRows=5) {
     this.numCols = numCols;
     this.numRows = numRows;
     this.data = {};
+    for (let row = 0; row < numRows - 1; row +=1) {
```
Comment (speschellk) [inline client/js/table-model.js]: It's unconventional to have for loops in a class constructor. Is there a different way you can achieve the same functionality?

File: client/js/table-view.js [hunk 2]
```diff
@@ -41,55 +43,178 @@ class TableView {
...
+    fragment.appendChild(tr);
+
     removeChildren(this.sheetBodyEl);
     this.sheetBodyEl.appendChild(fragment);
+
+  }
+  
+  renderColumnSum(col) {
+    let result = 0;
+    for(let i = 0; i < this.model.numRows - 1; i += 1) {
```
Comment (speschellk) [inline client/js/table-view.js]: Replace all += 1 instances with "++" (increments by one). Likewise, a -= 1 can be replaced with "--" (decrements by 1).

File: client/js/test/table-view.test.js [hunk 3]
```diff
@@ -11,6 +11,95 @@ describe('table-view', () => {
     document.documentElement.innerHTML = html;
   });
 
+
+
+  describe('add a row', () => {
+    it('should add a row', () => {
```
Comment (speschellk) [inline client/js/test/table-view.test.js]: What should? When? Make your test assertions descriptive of the functionality. A better one would be "if adds a row when the Add Row button is clicked", for example.

**Review by speschellk — CHANGES_REQUESTED** [review]

File: client/js/table-view.js (hunk 2 above)
Comment (michaeljclausen) [reply client/js/table-view.js]: Ok... but I was just going by what Doug Crockford said to do ;)

**Review by speschellk — APPROVED** [review]

**Pull request merged**